source/iostreams.tex

```diff
@@ -8152,7 +8152,7 @@
 Constructs an object of class
 \tcode{basic_ostringstream<charT, traits>},
 initializing the base class with
-\tcode{basic_ostream<charT, traits>(\&sb)}\iref{ostream}
+\tcode{basic_ostream<charT, traits>(addressof(sb))}\iref{ostream}
 and initializing \tcode{sb} with
 \tcode{basic_string\-buf<charT, traits, Allocator>(str, which | ios_base::out)}\iref{stringbuf.cons}.
 \end{itemdescr}
```

Comment:
And likewise here, it missed the `basic_ostream<charT, traits>::set_rdbuf(\&sb)` on line 8170.